.github/actions/build-single-project/action.yml

```diff
@@ -54,7 +54,9 @@ runs:
         if [ ! -e "/usr/local/bin/ninja" ]; then
           ln -s "/usr/bin/ninja" "/usr/local/bin/ninja"
         fi
-
+    - name: "Install Cmake"
+      shell: bash
+      run: echo "yes" | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --install "cmake;3.22.1" --channel=3
```

Comment:
`yes | sdkmanager --licenses` instead of `echo "yes"` if this is for licenses.

Reply:
so when i did that, it failed with "broken pipe". idk why i thought it sends too many "yes"s so i changed it to echo.